packages/rancher-operator-crd/package.yaml

```diff
@@ -0,0 +1,3 @@
+url: https://github.com/rancher/rancher-operator/releases/download/v0.2.0-alpha01/rancher-operator-crd-0.2.0-alpha01.tgz
```

Comment:
FYI @cbron the version here will show up in the Staging branch as `0.2.0-alpha0101-rc00` and in the Live branch as `0.2.0-alpha0101`. Is that okay?

If not, you might need to patch the version for each of these packages by running `make prepare`, making the change, and then running `make patch`.

Same comment for the other two charts

Reply:
fixed. Its now `-alpha01-rc01` as we want `-alpha01` to be final release.